recipe/meta.yaml

```diff
@@ -43,6 +43,7 @@ test:
   requires:
     - r-survival
     - r-ggplot2 >=2.2.1
+    - r-stats
```

Comment:
The "stats" package is part of the r-base distribution, if I'm not mistaken; so there is no `r-stats` package..

Reply:
Can you remove it and see how it goes?